pkg/deploy/strategy/recreate/recreate.go

```diff
@@ -110,9 +117,17 @@ func (s *RecreateDeploymentStrategy) DeployWithAcceptor(from *kapi.ReplicationCo
 		return fmt.Errorf("couldn't decode config from deployment %s: %v", to.Name, err)
 	}
 
+	retryTimeout := time.Duration(*config.Spec.Strategy.RecreateParams.TimeoutSeconds * int64(time.Second))
```

Comment:
Why not `time.Duration(*config.Spec.Strategy.RecreateParams.TimeoutSeconds)*time.Second`? Wrapping time.Second with int64 is ugly.